src/main/java/org/akaza/openclinica/control/core/SecureController.java

```diff
@@ -1598,9 +1588,79 @@ public int getSubjectCount(Study studyBean) {
         return subjectCount;
     }
 
-    protected  StudyBuildService getStudyBuildService(){
+    public Scheduler getSchemaScheduler(HttpServletRequest request, ApplicationContext context, Scheduler jobScheduler) {
```

Comment:
Instead of adding these methods here, can you create a base servlet for jobs and move these methods there? This class is already too big and I don't want us to add more logic here.